Assets/UI/MenuParticles/Particle/particle1.mat

```diff
@@ -0,0 +1,98 @@
+%YAML 1.1
```

Comment:
What is the use of the particle thing? What are they used for? I don't see any use cases in your changes